src/core/engines/Cesium/Feature/Box/hooks/box.ts

```diff
@@ -0,0 +1,294 @@
+import {
+  Cartesian3,
+  HeadingPitchRoll,
+  Quaternion,
+  Math as CesiumMath,
+  TranslationRotationScale,
+  Cartesian2,
+  Cartographic,
+} from "cesium";
+import { useCallback, useEffect, useMemo, useRef, useState } from "react";
+import { useCesium } from "resium";
+
+import { useContext } from "@reearth/components/molecules/Visualizer/Plugin";
```

Comment:
@rot1024 
How do I fix this API for plugin?
Can I keep importing old layer system until new plugin system is ported?